packages/core/admin/admin/src/render.ts

```diff
@@ -4,8 +4,16 @@ import { createRoot } from 'react-dom/client';
 
 import { StrapiApp, StrapiAppConstructorArgs } from './StrapiApp';
 
+interface FeatureConfig {
+  [key: string]: unknown;
+  future?: {
+    [key: string]: unknown;
```

Comment:
Lets add ones we know so we're keeping a live record of what we're supporting or not imo